src/batch-client/index.ts

```diff
@@ -1573,6 +1579,31 @@ export class ZimbraBatchClient {
 				});
 			}
 		});
+	
+
+		private downloadDoc = ({ id, url }: any) =>
```

Comment:
indentation

Reply:
how is this different then download method? only url?

Reply:
another weird thing I found downloadMessage passes isSecure as second argument, ideally it should be third argument, is this regression of the functionality?

Reply:
yes , only url structure is changing , as for messages we need to send part and we are generating url here ,
for documents i made the url and passed it , 
we can either  url for all methods and pass it  and keep a common function.

havent checked the downloadMessage functionality will have to check it